### PR TITLE
Feature/di movies linking tutorial ecc 4637

### PR DESCRIPTION
**Pull request by rpietzsch**
closes https://jira.brox.de/browse/ECC-4637

### CODE REVIEW

File: docs/build/create-project-with-linkingrule/index.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+
+!!! success "Step Result"
+
+    The project was created with the name "Movie links," which is displayed in the page top left corner.
+
+    ![image](22.2-project-created.png){ width="75%"}
+
+## 2. Upload the files
+
+-   Click on **add file** on the right side of the page.
```
Comment (seebi) [inline docs/build/create-project-with-linkingrule/index.md:74]: Its `Add File` on the screenshot.

File: docs/build/create-project-with-linkingrule/index.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+
+    ![image](22.2-click-on-add-files.png){ width="75%"}
+
+-   Click on **drop files here** select the file and click on **open**.
```
Comment (seebi) [inline docs/build/create-project-with-linkingrule/index.md:78]: Its `Drop files here or browse` on the screenshot.

Comment (seebi) [reply docs/build/create-project-with-linkingrule/index.md:78]: also `Open`

File: docs/build/create-project-with-linkingrule/index.md [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+
+!!! success "Step Result"
+
+    The files are uploaded to the project and reflect on the right side as shown below.
+
+    ![image](22.2-files-added.png){ width="75%"}
+
+## 3. Create datasets
+
+-   Click on **create.**
```
Comment (seebi) [inline docs/build/create-project-with-linkingrule/index.md:96]: - icon missing
- capitalisation wrong

(I will not mark missing icons or wrong capitalisation in the rest of the review)

File: docs/build/create-project-with-linkingrule/index.md [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+
+    ![image](22.2-click-on-create.png){ width="75%"}
+
+-   Click on the **dataset** on the left side in the item type and double-click on the **CSV file** in the centre.
```
Comment (seebi) [inline docs/build/create-project-with-linkingrule/index.md:100]: a double click? I am not aware of this - please validate.

File: docs/build/create-project-with-linkingrule/index.md [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+
+    ![image](22.2-files-uploaded.png){ width="75%"}
+
+-   Repeat the same step to add the file select the next second file and upload it.
+
+!!! success "Step Result"
+
+    The files are uploaded to the project and reflect on the right side as shown below.
+
+    ![image](22.2-files-added.png){ width="75%"}
```
Comment (seebi) [inline docs/build/create-project-with-linkingrule/index.md:92]: The files have two `.csv` suffixes. This is not exactly wrong but awkward. I would change it.

File: docs/build/create-project-with-linkingrule/index.md (hunk 4 above)
Comment (seebi) [reply docs/build/create-project-with-linkingrule/index.md:100]: ok, a double click is indeed possible.
But the default workflow is singe click + click on "Add" so I suggest to describe the default workflow.

File: docs/build/create-project-with-linkingrule/index.md [hunk 6]
```diff
@@ -0,0 +1,322 @@
...
+
+    ![image](22.2-type-dbpedia.png){ width="75%"}
+
+-   Tick the option **select file from project** and select the file from the drop-down list of files available in the project.
+
+    ![image](22.2-select-dbpedia-file.png){ width="75%"}
+
+-   Click on **create**.
+
+    ![image](22.2-click-on-create1.png){ width="75%"}
```
Comment (seebi) [inline docs/build/create-project-with-linkingrule/index.md:114]: The image is cropped wrongly so it looks ugly.

Comment (seebi) [reply docs/build/create-project-with-linkingrule/index.md:114]: I mean that the buttons are not completely seen and that there is a black line on the right.

File: docs/build/create-project-with-linkingrule/index.md [hunk 7]
```diff
@@ -0,0 +1,322 @@
...
+-   Click on **create**.
+
+    ![image](22.2-linking-click-on-create.png){ width="75%"}
+
+!!! success "Step Result"
+
+    Linking task is created for the project.
+    It shows the source path, target path, and operators (transform, comparison, and aggregation) options on the left side of the page.
+
+    ![image](22.2-linking-created.png){ width="75%"}
```
Comment (seebi) [inline docs/build/create-project-with-linkingrule/index.md:175]: badly cropped

File: docs/build/create-project-with-linkingrule/index.md [hunk 8]
```diff
@@ -0,0 +1,322 @@
...
+
+    It shows the linking editor, linking execution, linking evaluation, and reference link in the centre of the page to create and evaluate the linking rules.
+
+    ![image](22.2-linking-created.png){ width="75%"}
```
Comment (seebi) [inline docs/build/create-project-with-linkingrule/index.md:179]: badly cropped

File: docs/build/create-project-with-linkingrule/index.md [hunk 9]
```diff
@@ -0,0 +1,322 @@
...
+
+    On the right side of the page, it shows datasets, configuration linking, and configuration linking rules.
+
+    ![image](22.2-linking-created.png){ width="75%"}
```
Comment (seebi) [inline docs/build/create-project-with-linkingrule/index.md:183]: badly cropped

File: docs/build/create-project-with-linkingrule/index.md [hunk 10]
```diff
@@ -0,0 +1,322 @@
...
+
+    ![image](22.2-target-lowercase.png){ width="75%"}
+
+### Compare the `title` of the movies
+
+!!! success "Task"
+
+    Compare the movie title names of both the data `dbpedia.csv` and `linkedmdb.csv`.
+
+-   Click on **comparison** and type the **equality** in the search field and drag the `string equality` operation onto the canvas.
```
Comment (seebi) [inline docs/build/create-project-with-linkingrule/index.md:245]: `**comparison**` -> `**Comparison (:eccenca-operation-comparison:)**`

File: docs/build/create-project-with-linkingrule/index.md [hunk 11]
```diff
@@ -0,0 +1,322 @@
...
+
+-   Click on the down arrow button to **expand** all.
+
+    ![image](22.2-expand-all.png){ width="75%"}
+
+!!! success "Step Result"
+
+    This allows you to see how the linking rule was performed for each link and even the movie name compared in both files.
+
+    ![image](22.2-expand-result.png){ width="75%"}
```
Comment (seebi) [inline docs/build/create-project-with-linkingrule/index.md:293]: This screen has changed in 23.1 - please update.

File: docs/build/create-project-with-linkingrule/linkexecution.png
Comment (seebi) [inline docs/build/create-project-with-linkingrule/linkexecution.png:1]: remove lower windows bar and white screen.

File: docs/build/create-project-with-linkingrule/linking-evaluation.png
Comment (seebi) [inline docs/build/create-project-with-linkingrule/linking-evaluation.png:1]: editor menu is somehow under the main bar

File: docs/build/create-project-with-linkingrule/linking-result.png
Comment (seebi) [inline docs/build/create-project-with-linkingrule/linking-result.png:1]: There is an error sign ( top right (!) ) which indicates an error here. we should not have this on the screen

File: docs/build/create-project-with-linkingrule/linking-save.png
Comment (seebi) [inline docs/build/create-project-with-linkingrule/linking-save.png:1]: black line on the right could be removed

Comment (seebi) [reply docs/build/create-project-with-linkingrule/linking-save.png:1]: also badly cropped

File: docs/build/create-project-with-linkingrule/linking.png
Comment (seebi) [inline docs/build/create-project-with-linkingrule/linking.png:1]: This is nicely cropped ... :)

File: docs/build/create-project-with-linkingrule/lower-equality.png
Comment (seebi) [inline docs/build/create-project-with-linkingrule/lower-equality.png:1]: badly cropped on top

File: docs/build/create-project-with-linkingrule/save.png
Comment (seebi) [inline docs/build/create-project-with-linkingrule/save.png:1]: left menu cutted

File: docs/build/create-project-with-linkingrule/shaft.png
Comment (seebi) [inline docs/build/create-project-with-linkingrule/shaft.png:1]: What about the blue artifacts?

File: docs/build/create-project-with-linkingrule/source-path.png
Comment (seebi) [inline docs/build/create-project-with-linkingrule/source-path.png:1]: what is black artifact on the lower right?

File: docs/build/create-project-with-linkingrule/source-target-path.png
Comment (seebi) [inline docs/build/create-project-with-linkingrule/source-target-path.png:1]: what is black artifact on the lower right?

File: docs/build/create-project-with-linkingrule/transformation .png
Comment (seebi) [inline docs/build/create-project-with-linkingrule/transformation .png:1]: left menu cutted

File: docs/build/create-project-with-linkingrule/transformation1.png
Comment (seebi) [inline docs/build/create-project-with-linkingrule/transformation1.png:1]: left menu cutted

**Comment by seebi** [comment]
Most of the images have cropping issues. Please look into them. Some have artifacts, some are not on 23.1 but 22.2.

**Review by seebi — CHANGES_REQUESTED** [review]
Please see all my comments ...

File: docs/build/create-project-with-linkingrule/shaft.png
Comment (rpietzsch) [reply docs/build/create-project-with-linkingrule/shaft.png:1]: img not in use, pruned.

File: docs/build/create-project-with-linkingrule/source-target-path.png
Comment (rpietzsch) [reply docs/build/create-project-with-linkingrule/source-target-path.png:1]: img not in use, pruned

File: docs/build/create-project-with-linkingrule/transformation1.png
Comment (rpietzsch) [reply docs/build/create-project-with-linkingrule/transformation1.png:1]: img not in use, pruned

File: docs/build/create-project-with-linkingrule/source-path.png
Comment (rpietzsch) [reply docs/build/create-project-with-linkingrule/source-path.png:1]: img not in use, pruned

File: docs/build/create-project-with-linkingrule/index.md [hunk 12]
```diff
@@ -0,0 +1,322 @@
...
+
+-   [linkedmdb.csv](linkedmdb.csv)
+
+    !!! info
+
+        ![](preview-linkedmdb.png){ width="75%"}
+
+## 1. Create a new project
+
+-   Once logged in to Corporate memory on the right side click on **create.**
```
[the remark this answers is not in the document]
Comment (rpietzsch) [reply docs/build/create-project-with-linkingrule/index.md]: extended.

**Comment by seebi** [comment]
@rpietzsch How to proceed on this?

**Pull request closed without merging**